docs/src/components/productX/XPlans.tsx

```diff
@@ -38,7 +38,7 @@ export default function XPlans() {
                   Available with <GradientText>MIT and commercial licenses</GradientText>
                 </Typography>
               }
-              description="The advanced components are available under two licenses: MUI X, the MIT licensed, free to use version; and MUI X Pro, the commercially license version with the most advanced features."
+              description="The advanced components are available under two licenses: the MIT licensed, free to use version; and the commercially licensed: MUI X Pro and Premium, supporting the most advanced use cases."
```

Comment:
Revising to eliminate the nested colons and semi-colons.

```suggestion
              description="MUI X components are available under two licenses: MIT for the free community version, and commercial for Pro and Premium plans."
```

Reply:
Thanks, @samuelsycamore! Feel free to suggest an update for the Hero section too.